dist/docker/debian/build_docker.sh

```diff
@@ -26,7 +26,13 @@ version="$(<build/SCYLLA-VERSION-FILE)"
 release="$(<build/SCYLLA-RELEASE-FILE)"
 
 mode="release"
-arch="$(uname -m)"
+
+if uname -m | grep x86_64 ; then
+  arch="amd64"
+else
+  arch="arm64"
+fi
+
```

Comment:
This is a separate fix, please drop it from this patch, we can add it later.

Reply:
Actually, you need it because debian arch names are different from what uname -a returns.

Reply:
So change `else` to `elif` so we don't fail if someone runs it somewhere unexpected.

Reply:
Changed